src/cindex.jl

```diff
@@ -221,7 +221,11 @@ endof(tl::TokenList) = length(tl)
 done(tl::TokenList, i) = (i > length(tl))
 
 function getindex(tl::TokenList, i::Int)
-    if (i < 1 || i > length(tl)) throw(BoundsError()) end
+    if (i < 1 || i > length(tl))
+        # throw(BoundsError())
+        warn("TokenList miss at index ", i)
+        return Comment(string("TokenList miss at index ", i))
```

Comment:
Do you happen to have a backtrace when that happened? My guess would be wrap_c.jl:606 -- the logic there should probably check whether returned `pos` is over the length and avoid trying to index.

This change isn't going to break anything, but it might hide some preprocessor stuff that wouldn't have been processable anyway (the lexer is extremely minimal, just enough to handle `#define` constants and basic expressions, so it bails out and prints everything a comment).